subjects/devops/remake/README.md

```diff
@@ -0,0 +1,73 @@
+## remake
+
+### Instructions
+
+Create a file `remake.sh`, which will take one argument, the relative path of a directory, and will create new files and directories in it.
```

Comment:
Add something here to say that the perms, dates and names must be the same

Reply:
Done